lowell-dashboard/app/templates/profile.py

```diff
@@ -26,9 +26,9 @@
   </nav>
   
   <section id="bio">    
-    <p>{{user.first_name}}</p>
-    <p>{{user.last_name}}</p>
-    <p>{{user.email}}</p>
+    <p>First name: {{user.first_name}}</p>
+    <p>Last name: {{user.last_name}}</p>
```

Comment:
We just delete this? maybe add things like classes and login counts.

Reply:
Since we don't have them input it unless u want to do the same thing as the email where they agree to it and we will need to add it to the form for registration or create a specific form for that.

Reply:
It's ok. I might've added that by mistake. We don't need the first and last names.